### DISCUSSION TITLE
Can't See Code In Repo on Android Using Browser Interface on GitHub

### DISCUSSION THREAD

**Discussion by chtjfi** [community/community#86990] [Other Feature Feedback, Questions, & Ideas]
Today quite suddenly I can no longer see the code in my repos when in github.com (not the app) on Android.  It is the same for all repos.  I have always been able to view the code this way in the past.  Below a screenshot of what I see on the Android followed by what I see on a Windows laptop using the browser.

Has anyone seen this and perhaps knows what is wrong?

![Screenshot_20240113-163243.png](https://github.com/community/community/assets/24893509/edb8979a-ca1a-4daa-b966-d492ee6cefdd)

![IMG_20240113_163949_036.jpg](https://github.com/community/community/assets/24893509/0073f4d5-438a-4be7-871f-53b53628fb26)

**Comment by anuragjain-git** [comment]
It's working for me though, if you wait for few seconds code shows up, try refreshing the page or view it on desktop mode.

**Comment by m3047** [comment]
I got a similar report and am able to replicate on an old Macbook Air running El Cap. Neither Safari (old) or Firefox (relatively new) will display content for a repository _Code_ tab. If I know the URL I can still clone a repo. Other systems on this (IPv4) network work fine.

Was wondering if this was new policy or an anti-bot measure. Could impact people who aren't lucky enough to have first world problems. Do I need to host my repos somewhere else for the less fortunate?

**Comment by m3047** [comment]
See also #86584
